docs/streams/developer-guide/config-streams.html

```diff
@@ -708,6 +699,12 @@ <h4><a class="toc-backref" href="#id33">default.windowed.value.serde.inner</a><a
               <p>This is discussed in more detail in <a class="reference internal" href="datatypes.html#streams-developer-guide-serdes"><span class="std std-ref">Data types and serialization</span></a>.</p>
             </div></blockquote>
         </div>
+      <div class="section" id="windowed-inner-class-serde">
```

Comment:
We should add a deprecation hint, as KIP-1020 does deprecate this config, right?